website/src/pages/casestudies/index.md

```diff
@@ -0,0 +1,61 @@
+---
+slug: casestudies
```

Comment:
This needs updating. perhaps  borrowing the template from https://www.wgtwo.com/case-studies/